src/main/java/ai/elimu/web/content/word/WordCreateController.java

```diff
@@ -160,6 +171,44 @@ public String handleSubmit(
                 syllableDao.delete(syllable);
             }
             
+            // Generate Audio for this Word (if it has not been done already)
+            List<Audio> audios = audioDao.readAll(word);
+            if (audios.isEmpty()) {
+                Calendar timeStart = Calendar.getInstance();
+                Language language = Language.valueOf(ConfigHelper.getProperty("content.language"));
+                try {
+                    byte[] audioBytes = GoogleCloudTextToSpeechHelper.synthesizeText(word.getText(), language);
+                    logger.info("audioBytes: " + audioBytes);
+                    if (audioBytes != null) {
+                        Audio audio = new Audio();
+                        audio.setTimeLastUpdate(Calendar.getInstance());
+                        audio.setContentType("audio/mpeg"); // TODO: fetch from AudioFormat enum
+                        audio.setWord(word);
+                        audio.setTitle("word-" + word.getId());
+                        audio.setTranscription(word.getText());
+                        audio.setBytes(audioBytes);
+                        audio.setDurationMs(null); // TODO: Convert from byte[] to File, and extract audio duration
```

Comment:
Is it possible to copy/re-use the solution in the [AudioCreateController](https://github.com/elimu-ai/webapp/blob/master/src/main/java/ai/elimu/web/content/multimedia/audio/AudioCreateController.java#L140)?